google-cloud-graalvm-samples/graalvm-samples-client-library/pom.xml

```diff
@@ -23,5 +23,6 @@
     <module>spanner-sample</module>
     <module>storage-sample</module>
     <module>trace-sample</module>
+    <module>bigtable-sample</module>
```

Comment:
Recommend keeping this alphabetized unless there's a reason for the ordering.

Reply:
Done.